src/main/java/core/basesyntax/strategy/impl/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.OperationHandler;
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {

```

Comment:
The class `BalanceOperation` should not start with an empty line according to the checklist. Please remove the empty line at the beginning of the class implementation.